go/merge/three_way_test.go

```diff
@@ -19,16 +19,24 @@ type ThreeWayMergeSuite struct {
 	typeStr string
 }
 
-func (s *ThreeWayMergeSuite) tryThreeWayMerge(a, b, p, exp seq, vs types.ValueReadWriter) {
-	merged, err := ThreeWay(s.create(a), s.create(b), s.create(p), vs)
+func (s *ThreeWayMergeSuite) tryThreeWayMerge(a, b, p, exp seq, stores ...types.ValueReadWriter) {
```

Comment:
You could make this a little bit safer if instead of
`stores ...types.ValueReadWriter)`
you did
`store0 types.ValueReaderWriter, stores ...types.ValueReaderWriter)`
if you like.